ts/hooks/useHeaderSecondLevel.tsx

```diff
@@ -136,7 +136,9 @@ export const useHeaderSecondLevel = ({
             enableDiscreteTransition,
             animatedRef
           }
-        : {};
+        : {
+            ignoreSafeAreaMargin: !!alertProps
```

Comment:
Can we add `ignoreSafeAreaMargin: !!alertProps` directly in the return statement?